packages/react-vapor/src/components/refresh/RefreshCallbackWithButton.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {IReactVaporState} from '../../ReactVapor';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {Button, IButtonProps} from '../button/Button';
+import {RefreshCallBackActions} from './RefeshCallbackActions';
+import {IRefreshCallbackProps} from './RefreshCallback';
+import {RefreshStatus, RefreshStatusSelectors} from './RefreshCallbackReducer';
+
+export interface IRefreshCallbackWithButtonProps
+    extends IRefreshCallbackProps,
+        Partial<ReturnType<typeof mapStateToProps>>,
+        Partial<ReturnType<typeof mapDispatchToProps>> {}
```

Comment:
You don't need to export those Partial, check my comment on line 33. In fact in think you don't need this interface